src/rpm/builder.rs

```diff
@@ -747,6 +747,14 @@ impl RPMBuilder {
         let small_package = combined_file_sizes <= u32::MAX.into();
 
         let mut actual_records = vec![
+            IndexEntry::new(
+                IndexTag::RPMTAG_SOURCERPM,
+                offset,
+                IndexData::StringTag(format!(
+                    "{}-{}-{}.nosrc.rpm",
```

Comment:
Looking at RPM, this gets set to "(none)" by RPM internally when no headers are present. https://github.com/rpm-software-management/rpm/blob/master/lib/package.c#L284C1-L284C53

Maybe just using "(none)" is better?

Reply:
I also think now that "(none)" is better, thanks!

Reply:
Done!